src/librustc/metadata/decoder.rs

```diff
@@ -1536,3 +1538,12 @@ pub fn is_associated_type(cdata: Cmd, id: ast::NodeId) -> bool {
         Some(item) => item_sort(item) == 't',
     }
 }
+
+
+pub fn is_default_trait<'tcx>(cdata: Cmd, id: ast::NodeId) -> bool {
```

Comment:
Can we add a comment explaining what this is checking for? I haven't finished reading yet but i'm definitely a bit confused as to what a "default trait" is -- the name says trait, but it's treated like an impl.